tests/FunctionalTests/Persistence/ResetPasswordRequestRepositoryTest.php

```diff
@@ -10,6 +10,7 @@
 namespace SymfonyCasts\Bundle\ResetPassword\Tests\FunctionalTests\Persistence;
 
 use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\Common\Persistence\ObjectManager;
```

Comment:
Hmm, actually, this should use the `ObjectManager` from `doctrine/persistence` - this class is deprecated. I also see one more mention in `ResetPasswordTestFixtureRequestRepository.php`